runtime/ffi.h

```diff
@@ -15,7 +17,7 @@ struct C$FFI$CData: public refcountable_php_classes<C$FFI$CData<T>>, private Dum
   T c_value;
 
   const char *get_class() const noexcept { return "FFI\\CData"; }
-  int get_hash() const noexcept { return 1945543994; }
+  int get_hash() const noexcept { return static_cast<int32_t>(-1934403570); }
```

Comment:
Constant is changed. Is it safety?

Reply:
Yes, it has changed since the hashing algorithm was updated